hwy/detect_targets.h

```diff
@@ -107,8 +107,16 @@
 // Bit 38 reserved
 #define HWY_HIGHEST_TARGET_BIT_RVV 38
 
-// --------------------------- Future expansion: 4 targets
-// Bits 39..42 reserved
+// --------------------------- LoongArch: 3 targets (+ one fallback)
+// Bits 39 reserved (1 target)
+#define HWY_LASX (1LL << 40)
+#define HWY_LSX (1LL << 41)
+#define HWY_HIGHEST_TARGET_BIT_LOONG 41
+
+#define HWY_ALL_LOONGARCH (HWY_LASX | HWY_LSX)
```

Comment:
It is not necessary. On RVV we only have a single target, and x86 can simply check for <= because its targets are the lowest. This kind of macro can be useful for checking for "is it any LoongArch". I suppose that can also be done by checking the arch. We can remove this if you prefer.

Reply:
FYI there are now a couple of spots that write `(HWY_LASX | HWY_LSX)`. Depending on how likely you think that another target might be added in future, it could make sense to pull this into an ALL macro.